File: Api/WebApi/Models/Class.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebApi.Models
{
    public class Class
    {
        [Key]
        public int IdClass { get; set; }
        public int IdClassName { get; set; }
        public int IdClassLocation { get; set; }
        public DateTime Schedule { get; set; }
        public int MaxStudents { get; set; }
        public string StartTime { get; set; }
        public string FinishTime { get; set; }
        public float RegisterCost { get; set; }
        public bool Enabled { get; set; }
        public bool Spanish { get; set; }
        public Instructor instructor { get; set; }
        public string Note { get; set; }
        public string SearchKey { get; set; }
        public DateTime CreateDate { get; set; }
    }
}

File: TriSys/src/app/Models/Class.ts
export class ClassView {
    idClass: number = 0;
    idClassName: number = 0;
    className: string = '';
    idClassLocation: number = 0;
    classLocation: string = '';
    schedule: string = '';
    maxStudents: string = '';
    startTime: string = '';
    finishTime: string = '';
    registerCost: string = '';
    note: string = '';
    enabled: boolean = false;
    spanish: boolean = false;
    idInstructor: number = 0;
    instructor: string = '';
    searchKey: string = '';
    createDate: Date = new Date();
}
export class ClassInsert {
    Name: string = '';
    Location: string = '';
    Schedule: string = '';
    MaxStudents: string = '';
    StartDate: string = '';
    EndDate: string = '';
    RegisterCost: string = '';
    Note: string = '';
    Active: boolean = false;
    Spanish: boolean = false;
    Instructor: string = '';
    Private: boolean = false;
}
